bower.json

```diff
@@ -21,14 +21,14 @@
   ],
   "dependencies": {
     "polymer": "Polymer/polymer#0.8-preview",
-    "layout": "Polymer/layout",
-    "paper-styles": "PolymerLabs/paper-styles"
+    "paper-styles": "PolymerElements/paper-styles"
   },
   "devDependencies": {
     "webcomponentsjs": "Polymer/webcomponentsjs",
     "core-doc-viewer": "Polymer/core-doc-viewer#0.8-preview",
     "core-icon": "Polymer/core-icon#0.8-preview",
     "core-icons": "Polymer/core-icons#0.8-preview",
+    "test-fixture": "Polymer/test-fixture#master",
```

Comment:
You can currently use:

`polymerelements/test-fixture#^0.8.0`

And I have some PRs up (you are assigned) for other elements including the following:

```
polymerelements/iron-icon#^0.8.0
polymerelements/iron-icons#^0.8.0
```